crates/trippy-core/src/flows.rs

```diff
@@ -112,7 +112,7 @@ impl Flow {
     /// Check if a given `Flow` matches this `Flow`.
     ///
     /// Two flows are said to match _unless_ they contain different IP
-    /// addresses for the _same_ position (i.e. the `tll`).
+    /// addresses for the _same_ position (i.e. the `ttl`).
```

Comment:
As below, while `tll` could be a thing, nothing here indicates this is intended (unlike below, nothing precisely indicates that `ttl` is more correct, but I'm still guessing it is...)